tensorboard/plugins/graph/tf_graph_common/common_test.ts

```diff
@@ -192,5 +194,197 @@ describe('graph tests', () => {
         ])
       );
     });
+
+    it('groups numeric series of nodes without underscores', async () => {
```

Comment:
to codify the expected behavior for the future reader, add tests for:
- non-sequential numbering (e.g., 2 → 3 → 1 and 5 → 6 → 7 and 10 → 20 → 21) 
- mixed format (e.g., `foo_2` → `foo3` or `conv3` → `conv2` → `conv2_1`)
- any others? like `foo` → `bar` → `foo_2` → `foo3` → `bar_2`

Reply:
Added more test cases